base/scripts/code.notifications.cs

```diff
@@ -169,8 +169,7 @@
         %timeUntil2Min = ($Notifications::curTimeLeft - 120); //for example 179 - 120 = 59 seconds until 2 minute mark
 
         //this essentially bumps the main time check back on course at exactly 2 minutes
-        schedule("$curTimeAdjust = false;", %timeUntil2Min);
-        schedule("Game::checkTimeLimit();", %timeUntil2Min);
+        schedule("Notifications::Falsify();", %timeUntil2Min);
```

Comment:
i don't think this is worth a change - only worth pulling code into a function if its being called in multiple places otherwise it adds indirection making code slightly harder to follow. can you revert this change?

Reply:
i did this because im noticing issues with setting variables with the scheduler, not sure why but i feel like sometimes it is not 100%.... calling a function seems to work 100% of the time however, i found this out when i was testing another scheduler that needed something set to true and it was not issuing it, so I created a function with the only line to set the variable to true, and it worked.

Reply:
(i think this might be why sometimes the 2 minute marker gets screwed up) but only sometimes

Reply:
another time i saw you had this issue it was due to trying to send the variable name to the scheduler rather than the value of the variable. so this is what was being sent:

```
schedule("SomeFunction::(%variable);");
```

but it should be this to work:
```
schedule("SomeFunction::(" @ %variable @ ");");
```

i still think this is a code smell longer term and we need to go back to relying only on the original time limit checks - the suicide counter adds complexity and multiple conditions

Reply:
i tried this method and the results were the same, the only way ive been able to get it to work 100% of the time is by scheduling a function that then executes the commands i want.